decidim-elections/app/commands/decidim/elections/admin/update_election.rb

```diff
@@ -9,30 +9,54 @@ class UpdateElection < Decidim::Commands::UpdateResource
 
         def initialize(form, election)
           super
-          @attached_to = election
+          @attached_to = election unless election.published?
         end
 
-        protected
+        private
+
+        alias election resource
 
         def attributes
-          parsed_title = Decidim::ContentProcessor.parse(form.title, current_organization: form.current_organization).rewrite
-          parsed_description = Decidim::ContentProcessor.parse_with_processor(:inline_images, form.description, current_organization: form.current_organization).rewrite
+          election.published? ? published_election_attributes : unpublished_election_attributes
+        end
 
-          super.merge({
-                        title: parsed_title,
-                        description: parsed_description,
-                        start_at: form.manual_start ? nil : form.start_at,
-                        end_at: form.end_at,
-                        results_availability: form.results_availability
-                      })
+        def published_election_attributes
+          {
+            title: election.title,
+            description: parsed_description,
+            start_at: election.manual_start? ? nil : election.start_at,
+            end_at: election.end_at,
+            results_availability: election.results_availability
```

Comment:
```suggestion
            description: parsed_description
 ```
I think, if this is the only allowed attribute you can just ignore the others.
Btw, if you override the method "attributes", no need to specify the DSL `fetch_form_attributes :title, :description, :start_at, :end_at, :results_availability`